llvm/test/Transforms/InstCombine/ptrauth-intrinsics-call.ll

```diff
@@ -0,0 +1,76 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
```

Comment:
It's probably worth adding test cases for mismatching ptrauth parameters (key, discriminator, address discrimination) to ensure that combining is not performed when the parameters do not match. I like the approach for tests which is used in your other PR #94706.